react-native/react/native/remote-component-loader.js

```diff
@@ -6,6 +6,18 @@ import {ipcRenderer} from 'electron'
 
 const currentWindow = remote.getCurrentWindow()
 
+window.console.log = (...args) => {
+  ipcRenderer.send('console.log', args.join(' '))
```

Comment:
actually i don't think its the same. console.log allows you to have control strings

```
> console.log("%s-%s", 1, 2)
1-2
```

vs

```
> console.log("%s-%s", [1, 2].join(' '))
1 2-%s
```